packages/code-du-travail-frontend/pages/outils.js

```diff
@@ -3,7 +3,8 @@ import { withRouter } from "next/router";
 import getConfig from "next/config";
 import Head from "next/head";
 import fetch from "isomorphic-unfetch";
-import { Container, Alert } from "@cdt/ui";
+import { Container } from "@cdt/ui";
+import { Alert } from "@cdt/ui/esm";
```

Comment:
why introduce another folder since we have src, lib, es. 
what the difference with es ?